src/common.js

```diff
@@ -245,19 +245,21 @@ function randomFileName() {
 exports.randomFileName = randomFileName;
 
 // objectAssign(target_obj, source_obj1 [, source_obj2 ...])
-// Ponyfill for Object.assign
+// Polyfill for Object.assign
```

Comment:
nit: isn't this technically still a [ponyfill](https://ponyfoo.com/articles/polyfills-or-ponyfills) since it doesn't modify the global `Object`? Doesn't matter either way to me, just curious on the terminology. Either way, I agree with the implementation.